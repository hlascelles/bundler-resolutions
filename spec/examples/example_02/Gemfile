# frozen_string_literal: true

# This example will produce lock file without thor at all, as no other gem depends on it.

source "https://rubygems.org"

plugin "bundler-resolutions", path: "../../../"

gem "dememoize"

group :resolutions do
  gem "thor", "1.3.1"
end
