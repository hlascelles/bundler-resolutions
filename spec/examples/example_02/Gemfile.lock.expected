GEM
  remote: https://rubygems.org/
  specs:
    dememoize (0.1.0)

PLATFORMS
  ruby
  x86_64-linux

DEPENDENCIES
  dememoize

BUNDLED WITH
   2.5.14
